final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,181 @@
+import argparse
+import feedparser
+import logging
+import json
+import html
+import sys
+import datetime
+
+__version__ = '3.5'
+args = None
+limit = 3
+links = []
+print_setting = {
+    'feed': [
+        'title'
+    ],
+    'entries': [
+        'title',
+        'published',
+        'link',
+        'summary'
+    ]
+}
+
+
+def get_object_feed(url: str):
+    try:
+        data = feedparser.parse(url)
+        # if data.status == 200:
+        if data.bozo:
+            return f'There is no rss feed at this url: {url}'
+        else:
+            return data
+        # else:
+        #     return f'HTTP Status Code {data.status}'
+    except Exception as exc:
+        return f'ERROR: {exc}'
+
+
+def to_json(data, **kwargs):
+    # convert data to JSON format
+    result = {}
+    if 'feed' in kwargs.keys():
+        for i in kwargs['feed']:
+            result[i] = data['feed'][i]
+    if 'entries' in kwargs.keys():
+        result['items'] = []
+        for i in range(int(limit)):
+            temp = {}
+            for j in kwargs['entries']:
+                temp[j] = data['entries'][i][j]
+            result['items'].append(temp)
+    result = json.dumps(result)
+    return html.unescape(result)
+
+
+def get_img(input_string):
+    # from string type '<img src="link" alt="something">text' returns ('link', 'something')
```

Comment:
Here and below: docstrings are not comments, they should be like that:  `"""Text"""`